.github/workflows/pr-windows.yml

```diff
@@ -95,11 +95,20 @@ jobs:
       # See https://cmake.org/cmake/help/latest/manual/ctest.1.html for more detail
       run: ctest -C ${{env.BUILD_TYPE}} --verbose # --output-on-failure
 
-    - uses: actions/upload-artifact@v2
-      with:
-        name: coverage
-        path: ${{github.workspace}}/build/__coverage
     - uses: actions/upload-artifact@v2
       with:
         name: dist
         path: ${{github.workspace}}/build/dist
```

Comment:
@Pospelove BTW maybe move this before tests too? What do you thing? However... probably dist for a build that doesn't pass tests isn't much useful. (But logs for old PR runs get deleted anyway, so no big deal here, I think)